src/data/trainer-title-requirements.csv

```diff
@@ -10,7 +10,7 @@
 "育成でURAファイナルズ決勝に勝利しよう","Win the URA Finals in training"
 "育成で八大競走を全て勝利しよう","Win all eight major races in training"
 "クレーンゲームでぬいぐるみを\n10個以上獲得してクリアしよう","Win 10 or more stuffed toys in the Crane Game"
-"かけがえのない絆を\n感じるひとときを過ごそう","Spend a moment to feel an irreplaceable bond"
+"かけがえのない絆を\n感じるひとときを過ごそう","Win onsen tickets in the lottery, and then win URA Finals"
```

Comment:
```suggestion
"かけがえのない絆を\n感じるひとときを過ごそう","Win onsen tickets in the lottery, and then complete training"
```

Reply:
We'll get another scenario in the future so it's probably good to be vague

Reply:
That really depends on whether or not other scenarios will have the onsen scene, or if they'll even have the lottery - but sure, we can go with this for now